docs/source/manifest_validation.rst

```diff
@@ -0,0 +1,167 @@
+Validating a Metadata Manifest
+=================================================
+
+Requirements
```

Comment:
Could be good to follow a similar style as this PR: https://github.com/Sage-Bionetworks/schematic/pull/1584/files.  And add in links to the installation guide.